### PR TITLE
[2024-12-18] Make the number of online accesses in statistics collapsible

### PR DESCRIPTION
**Pull request by lunars97**
### Short description

Make the number of online accesses collapsible as a part of the #1097 as shown in [design proposal v3](https://www.figma.com/design/6U7R7Xj4wL7sbjxKRmOG9D/CMS-Project?node-id=1179-830&node-type=frame&t=vaCXaXccgqveytL3-0)

### Proposed changes

- Make use of the template for collapsible box
- Create a new file called statistics to hold all the upcoming collapsible separate boxes in statistics and the sidebar
- Change the path to the template in statistics view

### Side effects

- should be none?

### Resolved issues

Fixes: #3131 

__________________________________________________

[Pull Request Review Guidelines](https://digitalfabrik.github.io/integreat-cms/pull-request-review-guide.html)

### CODE REVIEW

**Review by JoeyStk — CHANGES_REQUESTED** [review]
Thank you for your PR! To me the boxes look at the moment like this 
grafik
This is not yet exactly what we want (s. design)
grafik.

As a general hint: I think you need to extract more parts from the `statistics_overview.html` file to the `statistics.html` file :)

**Comment by lunars97** [comment]
@JoeyStk I think I need to wait for this ticket #3170 in order to properly connect them, what do you think? or is it too much for this PR?

**Comment by JoeyStk** [comment]
> @JoeyStk I think I need to wait for this ticket https://github.com/digitalfabrik/integreat-cms/pull/3170 in order to properly connect them, what do you think? or will it be too much for this PR?

Yeah, that probably makes sense :)

**Review by JoeyStk — APPROVED** [review]
Thank you very much! It works as expected now :)
Thank you also for cleaning up the files! :) It's much appreciated :)

File: integreat_cms/cms/templates/statistics/statistics_sidebar.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+{% load widget_tweaks %}
+<div class="rounded border border-solid border-blue-500 shadow-2xl bg-white hidden mb-2"
+     id="chart-legend-container">
+    <div class="rounded p-4 bg-water-500">
+        <h3 class="heading font-bold text-black">
+            <i icon-name="settings" class="pb-1"></i> {% translate "Adjust shown data" %}
+        </h3>
+    </div>
+    <div id="chart-legend">
+        {% if request.region.statistics_enabled and perms.cms.view_statistics %}
```
Comment (charludo) [inline integreat_cms/cms/templates/statistics/statistics_sidebar.html:12]:
```suggestion
```
The same check is already performed in `statistics_overview.html`, right before importing this file.

Actually, maybe wrapping the entire contents of `statistics_overview.html` (i.e., everything in the `content` block) would be an option and allow you to remove all other checks?

**Review by charludo — APPROVED** [review]
Thanks, this looks and works great! I just have one nit (see below), where IMO we could remove a couple of the duplicate permission checks.

File: integreat_cms/cms/templates/statistics/statistics_sidebar.html (hunk 1 above)
Comment (lunars97) [reply integreat_cms/cms/templates/statistics/statistics_sidebar.html:12]: yes, you are right :+1: Thank you for bringing this up :)

**Pull request merged**